MTM-Frontend/src/pages/Auth/ForgotPassword.tsx

```diff
@@ -29,81 +27,94 @@ const ForgotPassword: React.FC = () => {
 
   useEffect(() => {
     if (currentUser) {
-      console.log("forgetpassword", currentUser);
+      console.log("forgot password", currentUser);
       navigate("/home");
     }
   }, [currentUser, navigate]);
 
   const {
     handleSubmit,
+    register,
     formState: { errors, isSubmitting },
+    control,
   } = useForm<FormValues>({
     resolver: yupResolver(schema),
   });
 
   const [message, setMessage] = useState<string>("");
   const [error, setError] = useState<string>("");
 
-  const[open, setOpen] = useState<boolean>(false);
+  const [open, setOpen] = useState<boolean>(false);
 
-  const onSubmit = async (values: FormValues) => {
+  const onSubmit = async (data: FormValues) => {
     try {
       setError("");
-      await forgotPassword(values.email);
-      setOpen(true)
+      await forgotPassword(data.email);
+      setOpen(true);
     } catch (err: any) {
       setError(err.message);
     }
   };
 
   return (
     <div className={"forgot-password-container"}>
-      <img className="forgot-logo-image" src={forgotpasswordlogo} alt="Image1" />
+      <img
+        className="forgot-logo-image"
+        src={forgotpasswordlogo}
+        alt="Image1"
+      />
       <Typography className="heading">Forgot your password?</Typography>
       <div className={"forgot-password-form"}>
         <form onSubmit={handleSubmit(onSubmit)}>
           <div className={"input-container"}>
             <p className={"info-text"}>
-              Enter your email below and receive your password reset instructions
+              Enter your email below and receive your password reset
+              instructions
             </p>
             <input
               type="email"
               placeholder="Email"
-              {...forgotPassword("email")}
+              {...register("email")}
```

Comment:
Any reason we changed forgotPassword method to register? This seems a bit anti-pattern